### PR TITLE
📦 NEW: Update Lerna to 2.8.0

### PR DESCRIPTION
**Pull request by ahmadawais**
Lerna 2.8.0 is a much better version and fixes a lot of problems that I personally faced. While this repo is using 2.0.0 — let's update this!

 — same PR as #70 @ntwb 👍

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (gziolo) [inline lerna.json]: @ntwb, we really need to add Eslint to make sure we don't need to deal with all that code formatting issues like tabs vs spaces.

File: lerna.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-	"lerna": "2.0.0",
-	"packages": [
-		"packages/*"
-	],
-	"version": "independent"
+  "lerna": "2.8.0",
```
Comment (gziolo) [inline lerna.json]: They use 2.0.0 in the official docs, it seems strange to have to update the version in both places. Shouldn't we reach out Lerna team and clarify that?

File: lerna.json (hunk 1 above)
Comment (ahmadawais) [reply lerna.json]: ESLint added at any stage as in after this PR would easily address these. I'll make a separate PR for that today :)

File: lerna.json (hunk 2 above)
Comment (ahmadawais) [reply lerna.json]: Nops! They recommend 2.x.x version.

Comment (ahmadawais) [reply lerna.json]: How did you initialize the lerna repo? You should run something like

Comment (gziolo) [reply lerna.json]: Actually lerna config was already here, that's why I had no idea how it should work :)

File: lerna.json (hunk 1 above)
Comment (gziolo) [reply lerna.json]: Yes, that's why I pinged Stephen.

Comment (ahmadawais) [reply lerna.json]: I have a working copy inside create-guten-block if you'd like to take a look at it?

File: lerna.json (hunk 2 above)
Comment (ahmadawais) [reply lerna.json]: Glad I could help. Let's merge this one :) 👋

File: lerna.json (hunk 1 above)
Comment (gziolo) [reply lerna.json]: 👌

Comment (ntwb) [reply lerna.json]: Yup, I'll take care of ESLint as stated in the other PR in the coming days

**Comment by ntwb** [comment]
Thanks @ahmadawais 👍

**Comment by ahmadawais** [comment]
Sure thing, @ntwb — would be contributing more. This is an exciting project for me.

**Comment by ntwb** [comment]
> They use 2.0.0 in the official docs, it seems strange to have to update the version in both places. Shouldn't we reach out Lerna team and clarify that?

@gziolo I've created https://github.com/lerna/lerna/issues/1237 asking to help clarify a few things for this

**Comment by gziolo** [comment]
@ntwb, thanks for doing that, it would be nice to have it simplified 👍

**Pull request merged**